sqlglot/dialects/dialect.py

```diff
@@ -137,6 +137,13 @@ def get_start_end(token_type: TokenType) -> t.Tuple[t.Optional[str], t.Optional[
         if enum not in ("", "bigquery"):
             klass.generator_class.SELECT_KINDS = ()
 
+        if enum not in ("", "hive", "spark", "spark2"):
```

Comment:
what about databricks?

Reply:
I missed the "various other clauses" part in their [docs](https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-qry-select.html) and assumed they didn't support these modifiers. Seems like I was wrong so gonna add that as well.